wiki/Help_centre/Account_restrictions/vi.md

```diff
@@ -23,22 +17,22 @@ Những chức năng sau sẽ bị tắt đối với tài khoản bị hạn ch
+Nếu tài khoản gốc của bạn chưa bị hạn chế, bạn có thể đăng nhập vào đó và tiếp tục chơi như bình thường, nhưng hãy đảm bảo rằng bạn sẽ không đăng nhập vào các tài khoản khác từ nay về sau, vì làm vậy sẽ chỉ gia tăng thêm hình phạt của bạn.
 
-Nếu khiếu nại xin gỡ hạn chế của bạn được chấp nhận và đang chơi tự nhiên bị hạn chế lại, đừng hoảng hốt! Hệ thống tự động có thể vô tình đánh dấu bạn, nhưng bạn sẽ được gỡ hạn chế trong 24 tiếng nếu đó thực sự là lý do. Hãy thoải mái gửi email hoặc tweet để chúng tôi để ý và gỡ hạn chế nhanh hơn.
+Nếu khiếu nại xin gỡ hạn chế của bạn được chấp nhận và đang chơi tự nhiên bị hạn chế lại, đừng hoảng hốt! Hệ thống tự động có thể vô tình đánh dấu bạn, nhưng bạn sẽ được gỡ hạn chế trong 24 tiếng nếu đó thực sự là lý do. Hãy thoải mái gửi email hoặc tweet để chúng tôi để ý và được gỡ hạn chế nhanh hơn.
 
 ## Tại sao tôi bị hạn chế?
 
 Có một vài lý do khiến một tài khoản bị hạn chế.
 
-- **Đó không phải là tài khoản đầu tiên của bạn.** Chơi nhiều tài khoản là phạm quy. Mọi tài khoản trừ cái đầu tiên sẽ bị hạn chế vĩnh viễn, nhưng tài khoản gốc ở hầu hết các trường hợp sẽ không bị làm sao, và chúng tôi mong bạn tiếp tục chơi bình thường. Nếu sự việc này nghiêm trọng hoặc do lạm dụng hệ thống, tài khoản gốc có thể bị Im lặng để cảnh cáo bạn tránh tạo thêm tài khoản, và nếu vẫn tiếp tục tái diễn thì tài khoản chính cũng sẽ bị hạn chế. Nếu tài khoản của bạn bị hạn chế và bạn biết mình còn tài khoản khác, xin đừng truy cập vào các tài khoản bị hạn chế mà hay quay về với cái gốc. Nếu bạn gặp vấn đề về việc truy cập hay xác thực, xin hãy gửi email tới [accounts@ppy.sh](mailto:accounts@ppy.sh) để chúng tôi có thể giúp bạn khôi phục lại nó.
-- **Bạn đã cho ai đó khác chơi trên tài khoản của bạn.** Chia sẻ tài khoản cũng là phạm quy. Nếu bạn cho ai đó dùng tài khoản của bạn, nó sẽ bị hạn chế, và người chơi sử dụng nó sẽ bị im lặng hay thậm chí là cũng bị hạn chế luôn. Nếu điều này xảy ra một cách vô ý, hãy cho chúng tôi biết càng sớm càng tốt qua email [accounts@ppy.sh](mailto:accounts@ppy.sh) .
+- **Đó không phải là tài khoản đầu tiên của bạn.** Chơi nhiều tài khoản là phạm quy. Mọi tài khoản trừ cái đầu tiên sẽ bị hạn chế vĩnh viễn, nhưng tài khoản gốc ở hầu hết các trường hợp sẽ không bị sao và chúng tôi mong bạn tiếp tục chơi bằng tài khoản đó. Nếu sự việc này nghiêm trọng hoặc do lạm dụng hệ thống, tài khoản gốc có thể bị im lặng để cảnh cáo bạn tránh tạo thêm tài khoản, và tài khoản chính sẽ bị hạn chế nếu việc này vẫn tiếp tục tái diễn. Nếu tài khoản của bạn bị hạn chế và bạn biết mình còn tài khoản khác, xin đừng truy cập vào các tài khoản bị hạn chế mà hay quay về với cái gốc. Nếu bạn gặp vấn đề về việc truy cập hay xác thực, xin hãy gửi email tới [accounts@ppy.sh](mailto:accounts@ppy.sh) để chúng tôi có thể giúp bạn khôi phục lại nó.
```

Comment:
Maybe be uniform - `tài khoản chính` is used instead of `tài khoản gốc` here. 

Maybe remove the subject altogether and reuse the former clause's one?
```suggestion
- **Đó không phải là tài khoản đầu tiên của bạn.** Chơi nhiều tài khoản là phạm quy. Mọi tài khoản trừ cái đầu tiên sẽ bị hạn chế vĩnh viễn, nhưng tài khoản gốc ở hầu hết các trường hợp sẽ không bị sao và chúng tôi mong bạn tiếp tục chơi bằng tài khoản đó. Nếu sự việc này nghiêm trọng hoặc do lạm dụng hệ thống, tài khoản gốc có thể bị im lặng để cảnh cáo bạn tránh tạo thêm tài khoản, và bị hạn chế nếu việc này vẫn tiếp tục tái diễn. Nếu tài khoản của bạn bị hạn chế và bạn biết mình còn tài khoản khác, xin đừng truy cập vào các tài khoản bị hạn chế mà hay quay về với cái gốc. Nếu bạn gặp vấn đề về việc truy cập hay xác thực, xin hãy gửi email tới [accounts@ppy.sh](mailto:accounts@ppy.sh) để chúng tôi có thể giúp bạn khôi phục lại nó.
```